src/main/java/uk/gov/companieshouse/filinghistory/consumer/apiclient/FilingHistoryApiClient.java

```diff
@@ -1,21 +1,28 @@
 package uk.gov.companieshouse.filinghistory.consumer.apiclient;
 
+import static uk.gov.companieshouse.filinghistory.consumer.Application.NAMESPACE;
+
 import java.util.function.Supplier;
+import org.apache.commons.lang3.StringUtils;
 import org.springframework.stereotype.Component;
 import uk.gov.companieshouse.api.InternalApiClient;
 import uk.gov.companieshouse.api.error.ApiErrorResponseException;
 import uk.gov.companieshouse.api.filinghistory.InternalFilingHistoryApi;
 import uk.gov.companieshouse.api.handler.exception.URIValidationException;
 import uk.gov.companieshouse.filinghistory.consumer.logging.DataMapHolder;
+import uk.gov.companieshouse.filinghistory.consumer.service.DeleteApiClientRequest;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
 
 @Component
 public class FilingHistoryApiClient {
 
     private static final String PUT_REQUEST_URI = "/company/%s/filing-history/%s/internal";
-    private static final String DELETE_REQUEST_URI = "/filing-history/%s/internal";
+    private static final String DELETE_REQUEST_URI = "/company/%s/filing-history/%s/internal";
 
     private final Supplier<InternalApiClient> internalApiClientFactory;
     private final ResponseHandler responseHandler;
+    private static final Logger LOGGER = LoggerFactory.getLogger(NAMESPACE);
```

Comment:
Should be up with the other static fields

Reply:
Resolved in latest commit